app/background.js

```diff
@@ -27,7 +27,7 @@ const checkToClose = function(cutOff) {
   }
 
   // Update the selected one to make sure it doesn't get closed.
-  chrome.tabs.getSelected(tabmanager.updateLastAccessed);
+  chrome.tabs.query({active: true}, tabmanager.updateLastAccessed);
```

Comment:
Oh yes good find, this might already avoid the race condition that I thought would be introduced. I will try to get this PR tested and merged.